src/Symfony/Bundle/SecurityBundle/Tests/DependencyInjection/CompleteConfigurationTest.php

```diff
@@ -448,6 +448,10 @@ public function testEncoders()
                 'class' => 'Symfony\Component\Security\Core\Encoder\BCryptPasswordEncoder',
                 'arguments' => array(15),
             ),
+            'JMS\FooBundle\Entity\User7' => array(
+                'class' => 'Symfony\Component\Security\Core\Encoder\Argon2iPasswordEncoder',
+                'arguments' => array(),
+            ),
```

Comment:
This is what breaks tests, as it runs on PHP 5.6. I suggest writing a dedicated test in SecurityExtensionTest, skipped if argon2i not supported.

Reply:
I'm not quite sure how to implement that - will need more details or be pointed in the right direction if I'm to implement that by end of play tomorrow, sorry!

Reply:
I did it for you in https://github.com/symfony/symfony/pull/21604/commits/51be4e50cacdc44f7dd65982ea281532acd581bc :)

Reply:
Awesome, thank you very much @chalasr! Looking at that commit, it would have taken me a long time to figure that out!